tests/ui/test_ui.py

```diff
@@ -16,6 +16,9 @@ def mock_external_classes(self, mocker):
     @pytest.fixture
```

Comment:
I'd consider the fixing of the incorrect keypress sizes to be a bugfix rather than (pure) refactor?

Reply:
Yeah, that would be appropriate. Thanks!